src/preFilter.ts

```diff
@@ -18,14 +19,18 @@ export function filterByContent(searchee: Searchee): boolean {
 		});
 	}
 
-	const isSingleEpisodeTorrent =
-		searchee.files.length === 1 && EP_REGEX.test(searchee.files[0].name);
-
-	if (!includeEpisodes && isSingleEpisodeTorrent) {
+	const isSingleEpisodeTorrent = searchee.files.length === 1 && EP_REGEX.test(searchee.name); 
```

Comment:
does this change carry a semantic difference? curious what you found in your testing here

Reply:
After discussion, @akesher123 mentioned utilizing name would yield more consistent/usable naming to test.

> also not sure on this but I think it might be better to use EP_REGEX.test(searchee.name) instead, only because I've seen some pretty terribly named files but torrent name is usually good

Simply took the advice...

I'll check this out further and post back.

Reply:
that sounds fine to me!

Reply:
Testing that I was able to do resulted in the following:

- All single file torrents yield identical results between files[0].name and .name. I don't have, or know how to consistently find, torrents that would yield differences in these values. I'd imagine they could exist, though. Probably on public trackers.

- Any torrents with a single season folder hierarchy resulted in searchee.name being the folder name, as opposed to file[0].name being the first file, such as S01E01.

- The only outlier I thought to test was something with multiple folders in the root torrent (multi-season packs) - but this also yielded the main torrent name (ex: Show S01-S05) with searchee.name (as opposed to any individual folders within) while files[0].name obviously returning the first file therewithin - such as S01E01.

Ultimately, my conclusion is that the results could affect whether EP_REGEX matches in extreme edge cases only (where the torrent name is actually MORE messed up than the file's naming). For actual single episodes, I was unable to find applicable outliers due to most of the sources I have being extremely consistent in the type of scenario. 

The one possible reason to utilize searchee.name would be for @akesher123's quoted reasoning, despite me being unable to actually find applicable torrents to run against. His logic does seem to track for those edge cases, wherever they are :P

Reply:
> All single file torrents yield identical results between files[0].name and .name. I don't have, or know how to consistently find, torrents that would yield differences in these values. I'd imagine they could exist, though. Probably on public trackers.

these exist. Most of the time, the difference between `.name` and `.files[0].name` is that the latter ends in `.mkv` and the former doesn't. Under the hood, these are distinct BitTorrent formats (single file torrents, which download as a file, and multi-file torrents, which download as a folder)

```js
{ 
  name: "Hello", 
  files: [
    { 
      name: "Hello.mkv", 
      path: "Hello/Hello.mkv" 
    }
  ]
}
```

```js
{ 
  name: "Hello.mkv", 
  files: [
    { 
      name: "Hello.mkv", 
      path: "Hello.mkv"
    }
  ]
}
```

However, I think for our purposes here, ^ is not an issue.


The rest all seems pretty sound. I like the change. 👍

Reply:
Understood. Appreciate the insight :)